src/main/java/com/mide/gangsaeng/bannedword/BannedWordServiceImpl.java

```diff
@@ -0,0 +1,42 @@
+package com.mide.gangsaeng.bannedword;
+
+import java.util.List;
+import java.util.concurrent.ConcurrentHashMap;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class BannedWordServiceImpl implements BannedWordService {
+
+    private static final ConcurrentHashMap<String,BannedWord> cache = new ConcurrentHashMap<>();
+
+    @Override
+    public boolean add(String word) {
+        if (isBanned(word)) {
+            return false;
+        }
+
+        cache.put(word, new BannedWord(word));
+        return true;
```

Comment:
Let's think about multi thread environment.
Can you think about the scenario if requests with same word comes in almost same time

Reply:
The final value is determined by the last request that was executed.
First request - `cache.put(word, new BannedWord(word))` added.
Second request - `cache.put(word, new BannedWord(word))` updated.

I want to distinguish it already has key scenario from not has key, so i used `putIfAbsent()`

Reply:
That's what I intended